### ISSUE TITLE
FeedDialog callback not called after post button clicked without internet connection

### ISSUE THREAD

**Issue by ava85** [Microsoft/winsdkfb#205]
Hi,

FeedDialog callback not called after post button clicked without internet connection. Bug repoduced in LoginCpp.Windows sample. In such cases facebook ios/android sdks closed dialog and called callback with erorr

**Comment by adiviness** [comment]
I'm not quite sure what you mean by callback, we don't implement a callback model in this sdk. We handle error reporting through `FBResult::Error`, which will contain an `FBError` object if the call wasn't successful. I tried to use the feed dialog button in our LoginCpp.Windows sample without an internet connection and I receive an `FBError` object that contains an error message stating "No internet". A dialog is never opened.

**Comment by ava85** [comment]
No, I mean click on post button in already opened dialog. In your case all fine

**Comment by ava85** [comment]
I try explain more clearly what I mean. FeedDialog called with internet connection and appeared ok. When I tap on "Post to Facebook" button inside FeedDialog with internet connection FacebookDialog::dialogWebView_FeedNavStarting called and it set dialogResponse. But when I tap on "Post to Facebook" button inside FeedDialog without internet connection - nothing happened. And after restoring internet connection i can't second time tap on "Post to Facebook" because it was disabled. 
The only way to interact with the dialogue remains close it.

**Comment by adiviness** [comment]
Ah, I see what you mean now. I was able to repro this, I'm not sure if there's a way for us to detect when this is happening. I need to do some more research.
